app/core/src/main/java/com/fsck/k9/job/K9IntervalMinutes.kt

```diff
@@ -0,0 +1,14 @@
+package com.fsck.k9.job
+
+/**
+ * Provides pre-defined time interval values in minutes
+ */
+class K9IntervalMinutes {
```

Comment:
K-9 Mail currently supports a lot more options. Dropping the ones below 15 minutes is fine. But we want to keep the rest. They live in `@array/check_frequency_values`. 
I think we can also get rid of this class. It should be enough to add constants for the default value and 'never' to `Account`.